app/views/candidate_interface/equality_and_diversity/edit_disabilities.html.erb

```diff
@@ -1,19 +1,31 @@
 <% content_for :title, title_with_error_prefix(t('equality_and_diversity.disabilities.title'), @disabilities.errors.any?) %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_edit_equality_and_diversity_disability_status_path) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_edit_equality_and_diversity_sex_path) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= form_with model: @disabilities, url: candidate_interface_edit_equality_and_diversity_disabilities_path, method: :patch do |f| %>
       <%= f.govuk_error_summary %>
 
       <%= f.govuk_check_boxes_fieldset :disabilities, multiple: false, caption: { text: t('equality_and_diversity.title'), size: 'xl' }, legend: { text: t('equality_and_diversity.disabilities.title'), size: 'xl', tag: 'h1' } do %>
+        <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+
+        <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
+
+        <p class="govuk-body">
+          You can <%= govuk_link_to 'ask for support as part of your application', candidate_interface_new_training_with_a_disability_path %> if you want to tell training providers about your needs straight away.</p>
```

Comment:
This link seems to take you to a page where your previous answer isn’t pre-populated, for some reason? I think maybe it needs to take you to the edit view?